.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Bug report
+about: Create a report to help us improve localtuya
+title: ''
+labels: ''
```

Comment:
I suggest that you add bug here.

Reply:
I was thinking about it, but as for the moment, we do not have a feature_request template, or a new_device template, should we do it know or automatically categorize once we have different templates ?

Reply:
Good point. Maybe we should just grab the bull by the horns and fix all of them? I have a few in my pyatv repo that has worked pretty well (for inspiration):

https://github.com/postlund/pyatv/tree/master/.github/ISSUE_TEMPLATE

Reply:
I had a go at a PR template in https://github.com/rospogrigio/localtuya/pull/782